snippets/ios/SampleProject/SampleProject/ViewController.swift

```diff
@@ -0,0 +1,417 @@
+//
+//  ViewController.swift
+//  SampleProject
+//
+//
+
+import UIKit
+import ThingIF
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+
+        let owner = Owner(TypedID(.thing, id: "dummyID"), accessToken: "dummyToken")
+        // CodeTagStart: variables
```

Comment:
tag name should be different with other platform, you can use like `variables_ios`